.github/workflows/windows-ci.yml

```
@@ -16,11 +16,24 @@ jobs:
 
```

probably need this too

```suggestion
    env:
      # https://github.com/actions/setup-dotnet/issues/122
      DOTNET_MULTILEVEL_LOOKUP: 1
```